src/components/Leaderboard.js

```diff
@@ -0,0 +1,37 @@
+//@flow strict
+
+import * as React from 'react';
+
+type Props = {

```

Comment:
nit: make these closed. (i. e. use vertical bars around them, as specified in the design doc)